src/components/ArticleList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useState, useEffect } from "react"
+import { fetchArticles } from "../../api"
+import ArticleCard from "./ArticleCard"
+
+function ArticleList() {
+
+  const [articles, setArticles] = useState([])
```

Comment:
We should have a loading state for any of the api requests we make to give the users visual feedback on slow load otherwise users will assume your app is broken and they'll navigate away